modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-test/src/testFunctional/tests/ProcessBuilderKaleoDesignerReact.testcase

```diff
@@ -201,12 +201,18 @@ definition {
 	}
 
 	@description = "Verify that the user can add Nodes into diagram by drag&drop: Start"
-	@ignore = "Test Stub"
 	@priority = "5"
 	test CanAddStartNode {
+		property test.name.skip.portal.instance = "ProcessBuilderKaleoDesignerReact#CanAddStartNode";
+
+		Workflow.openWorkflowListView();
 
-		// TODO CanAddStartNode pending implementation
+		Workflow.addNewDefinition(workflowDefinitionTitle = "New Workflow Definition");
 
+		ProcessBuilderKaleoDesignerReact.dragSidebarNodeAndMove(
+			nodeLabel = "Start",
+			nodePosition = "70,-200",
+			nodeType = "start");
 	}
```

Comment:
It would be interesting to add a `viewPanelNodePresent` to check the node in the diagram.